cmd/cerbosctl/inspect/policies.go

```diff
@@ -60,7 +60,7 @@ func (c *PoliciesCmd) Run(k *kong.Kong, cctx *client.Context) error {
 
 	tw := printer.NewTableWriter(k.Stdout)
 	if !c.Format.NoHeaders {
-		tw.SetHeader([]string{"POLICY ID", "POLICY", "ACTIONS", "VARIABLES"})
+		tw.SetHeader([]string{"POLICY ID", "ACTIONS", "VARIABLES"})
```

Comment:
The output should be sorted by store identifier now because it's the first column.